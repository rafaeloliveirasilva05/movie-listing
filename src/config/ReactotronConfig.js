import Reactotron from 'reactotron-react-js'

const tron = Reactotron
  .configure()
  .connect()

tron.clear()
console.tron = tron