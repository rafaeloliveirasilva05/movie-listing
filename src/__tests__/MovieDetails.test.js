import React from 'react'
import { mount, shallow } from '../config/enzyme'

import MovieDetails from '../pages/movieDetails'
import toJson from 'enzyme-to-json'

import { Provider } from 'react-redux'
import configureStore from 'redux-mock-store'


describe('screen elements test show movie', () => {

  const initialState = { movies: [] }
  const mockStore = configureStore()
  let store

  it('should render the homepage', () => {
    store = mockStore()
    const movieDetails = shallow(
      <Provider store={store}>
        <MovieDetails />
      </Provider>
    )

    expect(toJson(movieDetails)).toMatchSnapshot()
  })
})