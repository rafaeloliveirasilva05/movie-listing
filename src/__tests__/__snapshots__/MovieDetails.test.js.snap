// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`screen elements test show movie should render the homepage 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "clearActions": [Function],
        "dispatch": [Function],
        "getActions": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Subscription {
        "handleChangeWrapper": [Function],
        "listeners": Object {
          "notify": [Function],
        },
        "onStateChange": [Function],
        "parentSub": undefined,
        "store": Object {
          "clearActions": [Function],
          "dispatch": [Function],
          "getActions": [Function],
          "getState": [Function],
          "replaceReducer": [Function],
          "subscribe": [Function],
        },
        "unsubscribe": null,
      },
    }
  }
>
  <MovieDetails />
</ContextProvider>
`;
